src/LibraryManager.Contracts/PredefinedErrors.cs

```diff
@@ -92,5 +94,25 @@ public static IError PathOutsideWorkingDirectory()
         public static IError VersionIsNotSupported(string version)
            => new Error("LIB009", string.Format(Text.ErrorNotSupportedVersion, version));
 
+        /// <summary>
```

Comment:
@justcla : New strings that neeed review and translation. Can you please review and coordinate the translation? 
@prafullbhosale : looks like we are using the same codes for some of these! errors. Lets coordinate.